Apps/APN/ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don’t think we should push `GoogleService-Info.plist` to the repo. It contains sensitive information like the API key and should be added to `.gitignore`.

Reply:
`GoogleService-Info.plist` is required to build and upload sample app to Firebase, [otherwise it keeps failing](https://github.com/customerio/customerio-reactnative/actions/runs/10404077415/job/28811798606#step:17:6825).